client.go

```diff
@@ -225,12 +232,7 @@ func (co *Conn) ReadMsgHeader(hdr *Header) ([]byte, error) {
 		p = make([]byte, length)
 		n, err = io.ReadFull(co.Conn, p)
 	default:
-		if co.UDPSize > MinMsgSize {
-			p = make([]byte, co.UDPSize)
-		} else {
-			p = make([]byte, MinMsgSize)
-		}
-		n, err = co.Read(p)
+		fmt.Errorf("unknown connection type: %T", co.Conn)
```

Comment:
can this even happen?

Reply:
Unlikely as most connection will be catched by net.Conn anyway made that the default.